tcms/static/js/utils.js

```diff
@@ -1,17 +1,32 @@
 /*
     Used to update a select when something else changes.
 */
-function updateSelect(data, selector, id_attr, value_attr) {
+function updateSelect(data, selector, id_attr, value_attr, group_attr) {
     var _select_tag = $(selector)[0];
     var new_options = '';
 
-    // in some cases, e.g. TestRun search, the 1st <option> element is ---
+	currentGroup = '';
+	// check if using multiple select
+	isMultiple = _select_tag.attributes.getNamedItem('multiple') !== null;
+    
+	// in some cases using single select, the 1st <option> element is ---
     // which must always be there to indicate nothing selected
-    if (_select_tag.options.length) {
+    if (!isMultiple && _select_tag.options.length) {
         new_options = _select_tag.options[0].outerHTML;
     }
+	
+	data.forEach(function(element) {
```

Comment:
Do you mind keeping the indentations consistent across the file. Makes it a bit easier to follow the changes. At a later date we'll run eslint --auto-fix against this file anyway.

Reply:
Strangely neovim is auto-indenting and it looks consistent locally. Looks like an issue with my config, will fix